Project.toml

```diff
@@ -7,13 +7,14 @@ version = "1.0.0-DEV"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LogExpFunctions = "2ab3a3ac-af41-5b50-aa03-7779005ae688"
+Mmap = "a63ad114-7e13-5084-954f-fe012c677804"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 ProtoBuf = "3349acd9-ac6a-5e09-bcdb-63829b23a429"
 ReplMaker = "b873ce64-0db9-51f5-a568-4457d8e49576"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-julia = "1"
+julia = "1.5"
```

Comment:
this is to allow the keyword-argument syntax `f(; x)` as a shorthand as `f(; x=x)`, which I use below